Blockzilla/AppDelegate.swift

```diff
@@ -341,7 +342,8 @@ class AppDelegate: UIResponder, UIApplicationDelegate, ModalDelegate, AppSplashC
     
     func hideSplashView() {
         browserViewController.activateUrlBarOnHomeView()
-        splashView.animateHidden(true, duration: 0.25)
+        splashView.alpha = 0
+        splashView.isHidden = true
```

Comment:
`isHidden` should be enough. `alpha` is used usually in a context of an animation.

Reply:
Hey, @razvanlitianu, I tried first just to use `isHidden` property, but it won't work. If you use only `isHidden` the splash view won't appear at all. I think this is because we do have another method  `displaySplashAnimation` that is called at `application(_ application: UIApplication, didFinishLaunchingWithOptions...` and in that method we do have an animation that affects `alpha` property of Splash view. This is why we must also specify `alpha` value, otherwise it won't work properly.